### PR TITLE
Prepare tests for reactivation

### PR DESCRIPTION
**Pull request by marcoskwkm**
#### What is the purpose of this pull request?
This PR does a list of things so that they can be run and sent to Monitoring again.

- Delete videos of passing tests
The `cypress/videos` folder after a full test run would take over 1GB of disk space. This change erases the video of passing tests (they are not sent to Monitoring anyway) so the videos don't take much disk space, especially when running the tests locally.

- Send stable monitoring results to beta
We were sending `stable` tests to Monitoring's stable environment, while `beta` and `beta-io` (`vtexcommercestable` using workspace `beta`) were being sent to Monitoring's beta environment. This change sends all tests to the beta environment. This is because our tests might still have some issues that we need to fix and we don't want alarms triggering unnecessarily.

- Fix docker running tests on image creation
Self explanatory. The `dockerfile` had a `yarn test` instruction, which would run all tests. This probably explains the errors we were seeing when tests were run in Github.

- Fix Cypress version at 4.6.0
This is to avoid bugs due to Cypress changing versions.

- Disable test consistently failing due to API
This is mapped [here](https://app.clubhouse.io/vtex/story/32206/corrigir-teste-delivery-scheduled-delivery-and-delivery-2p) and hopefully will not be forgotten. We've already talked to the API folks about this, but this has not been fixed yet.

#### How should this be manually tested?
- Delete videos of passing tests
Run `yarn test:stable` and verify `cypress/videos/` only has videos of failed tests.

- Send stable monitoring results to beta
I don't think there's a way to test this before reactivating Actions in this repository.

- Fix docker running tests on image creation
Run `make ENVIRONMENT=stable`. Tests should run fine.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-IMAGE_NAME = vtex/checkout-ui-healthckeck-$(ENVIRONMENT)
+IMAGE_NAME = vtex/checkout-ui-healthcheck-$(ENVIRONMENT)
 
-deamon: build
-	docker run --ipc=host --shm-size 1024M -d $(IMAGE_NAME)
+daemon: build
+	docker run --ipc=host -it --shm-size 1024M $(IMAGE_NAME) yarn test:$(ENVIRONMENT)
```
Comment (lucasecdb) [inline Makefile:4]: does this still run in the background?

Comment (marcoskwkm) [reply Makefile:4]: No. Should it? I really don't know if this makes a difference when running in CI, and when running locally I'd like to see the output of the tests in the console.

Comment (lucasecdb) [reply Makefile:4]: hmmm makes sense, can we rename the script from `daemon` to `run` then?

Comment (marcoskwkm) [reply Makefile]: Sure, done.

**Comment by marcoskwkm** [comment]
Release commit.

**Pull request merged**